src/main/scala/com/libertyglobal/odh/kafkatool/Main.scala

```diff
@@ -34,24 +36,89 @@ import scala.reflect.io.File
 
 object Main extends StrictLogging {
 
-  def sortedToString[T: Ordering](xs: Iterable[T]): String = {
-    xs.toSeq.sorted.mkString(",")
-  }
+  def main(args: Array[String]): Unit = {
```

Comment:
how come that the whole file is changed now?

Reply:
I made the `Reformat code` thing, haven't noticed it's a mess. I'll arrange it